### ISSUE TITLE
Create a Google docs plugin/app

### ISSUE THREAD

**Issue by suchow** [amperser/proselint#14]

**Comment by danielnaber** [comment]
There's an unmaintained prototype for LanguageTool which could maybe be used: https://github.com/jordimas/google-docs-language-tool

**Comment by danielnaber** [comment]
LanguageTool now has an add-on that could probably be adapted: https://github.com/jaumeortola/languagetool-google-docs

**Comment by milekpl** [comment]
I think this may be closed, the plugin works :)

**Comment by suchow** [comment]
@milekpl I'm unaware of a Google Docs plugin — can you provide a link?

**Comment by danielnaber** [comment]
Source: https://github.com/jaumeortola/languagetool-google-docs
Store: https://chrome.google.com/webstore/detail/languagetool/kjcoklfhicmkbfifghaecedbohbmofkm

**Comment by suchow** [comment]
@danielnaber I know about the LanguageTool plugin, but I think @milekpl must be referring to one for Proselint? (Unless he's posted this comment on the wrong repo?)

**Comment by milekpl** [comment]
Yeah, I meant the LanguageTool plugin. I thought it was an issue in our repo. Sorry for any confusion.
